CONTRIBUTING.md

```diff
@@ -0,0 +1,171 @@
+# Contributing guidelines
+
+We adhere to the [Contributor Covenant Code of Conduct](#contributor-covenant-code-of-conduct) found below.
+
+### To add a new feature
+
+- fork to your private github
```

Comment:
Good. The idea is not to make an exact step-by-step but mostly GUIDELINES . Take into account that we change very fast and cannot afford to update all doc at the same time. Stick to the rationale behind things and delegate details to official doc sites. Therefore, I would add to your text links to all relevant github doc e.g.

 [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) to your private github.

This way you keep here the "eagle's view" and there you can find step by step indications.